package org.opm.busybeaver.exceptions.service;

public class TeamAlreadyExistsForUserException {
}
